### PR TITLE
Miscellaneous typos

### PR DESCRIPTION
**Pull request by herbelin**
**Kind:** typos

These are a few typos found in passing (in doc, comments, or anomaly message).

### CODE REVIEW

File: doc/sphinx/addendum/universe-polymorphism.rst [hunk 1]
```diff
@@ -529,8 +529,8 @@ sections, except in the following ways:
        Polymorphic Universe i.
        Fail Constraint i = i.
 
-  This includes constraints implictly declared by commands such as
-  :cmd:`Variable`, which may as a such need to be used with universe
+  This includes constraints implicitly declared by commands such as
+  :cmd:`Variable`, which may as such need to be used with universe
```
Comment (herbelin) [inline doc/sphinx/addendum/universe-polymorphism.rst]: Something had to be fixed, but not fully sure about the correct fix ("which may thus require to be used ..."?).

**Review by ppedrot — APPROVED** [review]

File: doc/sphinx/addendum/universe-polymorphism.rst (hunk 1 above)
Comment (jfehrle) [reply doc/sphinx/addendum/universe-polymorphism.rst]: Seems fine.  I think you could drop "as such".  I don't think it adds anything.

**Review by jfehrle — APPROVED** [review]
Looks good.  One suggestion below.

**Review by SkySkimmer — APPROVED** [review]

File: doc/sphinx/addendum/universe-polymorphism.rst (hunk 1 above)
Comment (herbelin) [reply doc/sphinx/addendum/universe-polymorphism.rst]: I was also thinking at dropping "as such" but kept it in doubt.

I made the change, thanks.

**Review by Zimmi48 — APPROVED** [review]

**Pull request merged**